production/config.yml.dockerfile

```diff
@@ -158,12 +158,16 @@ development: &default
   #
   google_analytics_code: # '<GOOGLE ANAYTICS CODE>'
 
-  # Honeycomb analytics service https://www.honeycomb.io
+  # Open Telemetry configuration, for reporting to Honeycomb
```

Comment:
@caiosba could you help me understand what this file does and what its relationship to config.yml.example should be? I wasn't sure if we need to keep them in lockstep or what.

Reply:
I didn't remember that this file even existed... I don't think it's used in our live deployments anymore, but @sonoransun can confirm.

Reply:
I see it's being called in `production/Dockerfile`. but it seems to be [deleted in the same line it's created](https://github.com/meedan/check-api/blob/develop/production/Dockerfile#L68)? Also realize I'm not sure how that Dockerfile is being used

Reply:
This is using a default config to generate the precompiled assets, E.g. it's all in service to this command: `bundle exec rake assets:precompile`. Note that these steps are done when building the image, but the final entrypoint command, `ENTRYPOINT ["/opt/bin/start.sh"]`, isn't invoked until runtime. At this point, proper QA or Live configs will be generated as expected.

I think it would be better to copy the default config into place, removing this config entirely. E.g. use this line instead: `RUN cp config/config.yml.example config/config.yml && \`

Thanks for catching these @hartsick , it's easy to overlook them otherwise!

Reply:
awesome! thanks for that context, @sonoransun ! I just wanted to make sure I wasn't missing any configs we needed to set - think I found this initially when cmd + f'ing